packages/lit-dev-content/site/tutorials/content/working-with-lists/00.md

```diff
@@ -0,0 +1,13 @@
+In this tutorial, we will be exploring different ways of rendering a list of
+items in Lit!
+
+Let's begin by looking a how we might render a list of names from an array.
+We've already set up a custom element with a `names` property that is an
+array of strings.
+
+We can utlize the `map` array prototype method to generate a list item
```

Comment:
Perhaps before going into the first action for the user add a section on "prior knowledge: docs link docs link docs link"

like reactive properties. We expect you to know how reactive properties work and handle updates

Reply:
Good idea. How about also adding like a recommend prerequisite tutorial if applicable?

Something like
> Prior knowledge: [Defining a custom element, rendering a template, and reactive properties](http://localhost:5415/docs/components/overview/).
> Recommended prerequisite: [Intro to Lit](https://lit.dev/tutorials/intro-to-lit/).

to be placed under the introductory description of the tutorial.

Reply:
That's the idea! Hopefully once we get a bunch of tutorials up i can go through and link to each other